app/controllers/window/clients/files_controller.rb

```diff
@@ -18,6 +19,8 @@ def index
 
       @consent_files = consent_scope
       @files = file_scope.page(params[:page].to_i).per(20).order(created_at: :desc)
+      @deleted_files = all_file_scope.only_deleted
```

Comment:
Pagination isn't working correctly.  It doesn't appear to paginate consent files (which may be fine) but when the other files are paginated, clicking to page two jumps you back to the consent files.